subgen.py

```diff
@@ -504,13 +510,25 @@ async def asr(
         delete_model()
     
     if result:
-        return StreamingResponse(
-            iter(result.to_srt_vtt(filepath=None, word_level=word_level_highlight)),
-            media_type="text/plain",
-            headers={
-                'Source': 'Transcribed using stable-ts from Subgen!',
-            }
-        )
+        if output == 'srt':
+          return Response(
+              content=result.to_srt_vtt(filepath=None, word_level=word_level_highlight, segment_level=segment_level),
+              media_type="text/plain",
+              headers={
+                  'Source': 'Transcribed using stable-ts from Subgen!',
+              }
+          )
+        elif output == 'json':
```

Comment:
Not opposed to implementing, what's the usecase?

Reply:
bleh, sorry for the mess up. I pushed changes to my main unrelated to this PR. I've reverted to the original changeset. That'll teach me to use a branch on my fork. But to answer the question, I am doing some post-processing using the word timings so having the raw result JSON is handy for that.